app/views/additional_payments/claims/poor_performance.html.erb

```diff
@@ -7,6 +7,7 @@
       "subject_to_disciplinary_action": "claim_subject_to_disciplinary_action_true"
     }) if @form.errors.any? %>
     <%= form_for @form, url: claim_path(current_journey_routing_name) do |form| %>
+      <%= form.hidden_field :journey_session_id, value: journey_session.id %>
```

Comment:
I think we probably want to pass this into the form object's constructor rather than pass it as a hidden field, which will be much easier to do once this pr https://github.com/DFE-Digital/claim-additional-payments-for-teaching/pull/2736 is merged